android/src/main/java/io/sentry/react/RNSentryModuleImpl.java

```diff
@@ -615,6 +625,46 @@ public void disableNativeFramesTracking() {
         }
     }
 
+    public void startProfiling(Promise promise) {
+        final WritableMap result = new WritableNativeMap();
+        try {
+            RNSentryModuleImpl.javaScriptExecutorFactory.startSamplingProfiler();
+        } catch (UnsupportedOperationException e) {
+            String error = javaScriptExecutorFactory.toString() + "does not support Sampling Profiler";
+            logger.log(SentryLevel.ERROR, error);
+            result.putString("error", error);
```

Comment:
L: If we log the error in the Native layer, we probably don't need to send the error back?

Reply:
Make sense to log only one, let's do it in the JS layer for users' convenience, so it's easier to see the log.